### PR TITLE
Users/savenkat/external m150

### PR DESCRIPTION
**Pull request by savenkat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
   ],
   "dependencies": {
     "@kubernetes/client-node": "~0.8.1",
-    "azure-devops-ui": "1.149.3",
+    "azure-devops-ui": "^1.149.5",
```
Comment (madhuv-msft) [inline package.json]: we can make this as 1.149.5 if needed

File: webpack.config.js [hunk 2]
```diff
@@ -61,8 +61,23 @@ module.exports = {
     child_process: 'empty',
     net: 'empty'
   },
-  externals: {
-    react: 'react',
-    "react-dom": 'react-dom'
-  }
+  externals: [
```
Comment (madhuv-msft) [inline webpack.config.js]: how are we making sure other public package is built?

File: webpack.config.js [hunk 3]
```diff
@@ -61,8 +61,23 @@ module.exports = {
...
-  externals: {
-    react: 'react',
-    "react-dom": 'react-dom'
-  }
+  externals: [
+    function(context, request, callback) {
+      if (request === "react" || request === "react-dom") {
+        return callback(null, request);
+      }
+      else if (request.startsWith("azure-devops-ui/Core")) {
```
Comment (madhuv-msft) [inline webpack.config.js]: "azure-devops-ui" --> should we make this as const

Comment (savenkat) [reply webpack.config.js]: will fix in next PR

File: package.json (hunk 1 above)
Comment (savenkat) [reply package.json]: done

File: webpack.config.js (hunk 2 above)
Comment (savenkat) [reply webpack.config.js]: we will publish it from master branch for the time being

**Pull request merged**